providers/standard/src/airflow/providers/standard/sensors/time.py

```diff
@@ -42,7 +42,10 @@ class StartTriggerArgs:  # type: ignore[no-redef]
         timeout: datetime.timedelta | None = None
 
 
-from airflow.utils import timezone
+try:
+    from airflow.sdk import timezone
```

Comment:
This isn't right, you should be using:
```
from airflow.sdk._shared.timezones import timezone
```

Reply:
Why should I be accessing a protected member of the module?